### ISSUE TITLE
RuntimeError: An Exception occurred, aborting execution of iterator: lambda1() takes exactly one argument (4 given)

### ISSUE THREAD

**Issue by gijzelaerr** [networkit/networkit#494]
hi!

I think i found a bug.

If I run this Python code:
```python
import networkit as nk
g = nk.graph.Graph(directed=True)
[...] # populate graph
g.inNeighbors(23424)
```

I get the error:
```
networkit/_NetworKit.pyx in networkit._NetworKit.Graph.inNeighbors()
networkit/_NetworKit.pyx in networkit._NetworKit.Graph.forInEdgesOf()
RuntimeError: An Exception occurred, aborting execution of iterator: lambda1() takes exactly one argument (4 given)
```

Or am I doing something wrong? I'm running NetworKit 6.0 on Linux (Ubuntu 19.10) with Python 3.7.5.

**Comment by angriman** [comment]
Hi, yes that's indeed a bug. We solved it in #470 but we did not release a new version with the fix yet. If you need to use that feature you can clone the repository and build it from source (i.e., cd to the root directory, and type `pip3 install -e .`)

**Comment by gijzelaerr** [comment]
ah perfect! Thanks. Sorry for missing that report.
